src/lib.rs

```diff
@@ -1,10 +1,10 @@
 //! # base16_color_scheme
 //!
-//! A library to build [base16](https://github.com/chriskempson/base16) colorschemes written in Rust.
+//! A library to build [base16](https://github.com/tinted-theming/variables) colorschemes written in Rust.
```

Comment:
I guess this is just a typo and should lead to <https://github.com/tinted-theming/home>, though I'm not sure.

Reply:
Yeah this was a mistake for this path
```suggestion
//! A library to build [base16](https://github.com/tinted-theming/home) colorschemes written in Rust.
```